lm_eval/models/huggingface.py

```diff
@@ -667,6 +667,8 @@ def _detect_batch_size(self, requests=None, pos: int = 0):
             max_cont_enc = len(continuation_enc[-(self.max_length + 1) :])
         else:
             max_length = self.max_length
+            max_context_enc = max_length
+            max_cont_enc = max_length
```

Comment:
Should we always assume max_length will refer to both the encoder and decoder lengths?

Reply:
Not necessarily but having a user specify several max lengths seems annoying--does HF expose or note the enc and dec max lengths in configs anywhere / is this at all detectable?

Reply:
Not to my knowledge. In that case I suppose this is good enough.